internal/server/handlers.go

```diff
@@ -103,6 +117,141 @@ func (s *Server) shortenJSON(w http.ResponseWriter, r *http.Request) {
 	s.writeResponse(s.setHeader(w, err), response)
 }
 
+func (s *Server) removeURLs(w http.ResponseWriter, r *http.Request) {
+	s.Log.Info("Получили запрос на удаление ссылок")
+	var reqJSON []string
+	// получаем JSON
+	requestBody, err := getURLJSON(w, r, reqJSON)
+	if err != nil {
+		http.Error(w, "Ошибка получения JSON: "+err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	doneCh := make(chan struct{})
+
+	inputCh := s.generator(doneCh, requestBody)
```

Comment:
SUGGESTION: it should be in another layer.

Reply:
Done